### PR TITLE
feat: remove cupertino

### PR DESCRIPTION
**Pull request by arthurbleilzup**
Remove cupertino as style of components, keeping only Material Design.

### CODE REVIEW

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: beagle_components/lib/src/internal/beagle_refresh_indicator.dart [hunk 1]
```diff
@@ -255,25 +257,29 @@ class BeagleRefreshIndicatorState extends State<BeagleRefreshIndicator> with Tic
         break;
     }
     if (indicatorAtTopNow != _isIndicatorAtTop) {
-      if (_mode == _RefreshIndicatorMode.drag || _mode == _RefreshIndicatorMode.armed)
+      if (_mode == _RefreshIndicatorMode.drag ||
+          _mode == _RefreshIndicatorMode.armed)
         _dismiss(_RefreshIndicatorMode.canceled);
```
Comment (botbeaglezup) [inline beagle_components/lib/src/internal/beagle_refresh_indicator.dart:262]: <table data-meta="generated_by_flutter">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-flutter/blob/2968c94093a0122155d34b252914a53a9ee82749/beagle_components/lib/src/internal/beagle_refresh_indicator.dart#L262"></span>DO use curly braces for all flow control structures</td>
    </tr>
  </tbody>
</table>

File: beagle_components/lib/src/internal/beagle_refresh_indicator.dart [hunk 2]
```diff
@@ -327,11 +332,14 @@ class BeagleRefreshIndicatorState extends State<BeagleRefreshIndicator> with Tic
...
 
   void _checkDragOffset(double containerExtent) {
-    assert(_mode == _RefreshIndicatorMode.drag || _mode == _RefreshIndicatorMode.armed);
-    double newValue = _dragOffset / (containerExtent * _kDragContainerExtentPercentage);
+    assert(_mode == _RefreshIndicatorMode.drag ||
+        _mode == _RefreshIndicatorMode.armed);
+    double newValue =
+        _dragOffset / (containerExtent * _kDragContainerExtentPercentage);
     if (_mode == _RefreshIndicatorMode.armed)
       newValue = math.max(newValue, 1.0 / _kDragSizeFactorLimit);
```
Comment (botbeaglezup) [inline beagle_components/lib/src/internal/beagle_refresh_indicator.dart:340]: <table data-meta="generated_by_flutter">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-flutter/blob/2968c94093a0122155d34b252914a53a9ee82749/beagle_components/lib/src/internal/beagle_refresh_indicator.dart#L340"></span>DO use curly braces for all flow control structures</td>
    </tr>
  </tbody>
</table>

File: beagle_components/lib/src/internal/beagle_refresh_indicator.dart [hunk 3]
```diff
@@ -327,11 +332,14 @@ class BeagleRefreshIndicatorState extends State<BeagleRefreshIndicator> with Tic
...
-    _positionController.value = newValue.clamp(0.0, 1.0); // this triggers various rebuilds
+    _positionController.value =
+        newValue.clamp(0.0, 1.0); // this triggers various rebuilds
     if (_mode == _RefreshIndicatorMode.drag && _valueColor.value.alpha == 0xFF)
       _mode = _RefreshIndicatorMode.armed;
```
Comment (botbeaglezup) [inline beagle_components/lib/src/internal/beagle_refresh_indicator.dart:344]: <table data-meta="generated_by_flutter">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-flutter/blob/2968c94093a0122155d34b252914a53a9ee82749/beagle_components/lib/src/internal/beagle_refresh_indicator.dart#L344"></span>DO use curly braces for all flow control structures</td>
    </tr>
  </tbody>
</table>

File: beagle_components/lib/src/internal/beagle_refresh_indicator.dart [hunk 4]
```diff
@@ -385,18 +397,16 @@ class BeagleRefreshIndicatorState extends State<BeagleRefreshIndicator> with Tic
             FlutterError.reportError(FlutterErrorDetails(
```
Comment (botbeaglezup) [inline beagle_components/lib/src/internal/beagle_refresh_indicator.dart:397]: <table data-meta="generated_by_flutter">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-flutter/blob/2968c94093a0122155d34b252914a53a9ee82749/beagle_components/lib/src/internal/beagle_refresh_indicator.dart#L397"></span>DO use curly braces for all flow control structures</td>
    </tr>
  </tbody>
</table>

**Review by matheusribeirozup — APPROVED** [review]

**Pull request merged**